core/src/verification/queue/kind.rs

```diff
@@ -104,8 +103,8 @@ pub mod headers {
             *self.parent_hash()
         }
 
-        fn score(&self) -> U256 {
-            *self.score()
+        fn block_number(&self) -> U256 {
```

Comment:
BTW, we are using the `BlockNumber` type for block numbers which is an alias type of `u64`. Do we need to use the looser type `U256` ?

Reply:
These functions seem unnecessary and are unused, so I removed these functions. Thank you : )

Reply:
I think you should remove the function from the original commit not remove the logic in the last commit.

Reply:
You mean remove the function from the commit `Remove the logic related to the score in queue` commit?

Reply:
Yes, unnecessary changes don't have to exist in commit histories.

Reply:
I modified it : ) Thank you for your good advice!